packages/api/src/external/commonwell/command/cw-patient-data/update-cw-data.ts

```diff
@@ -78,3 +91,11 @@ export async function updateCwPatientDataWithinDBTx(
     { transaction }
   );
 }
+
+function isContainedAt(link: CwLink, linksToInvalidate: CwLink[]): boolean {
+  const linkOid = getLinkOid(link);
+  const isInvalid = linksToInvalidate.some(function (invalidLink) {
+    return getLinkOid(invalidLink) === linkOid;
+  });
+  return !isInvalid;
+}
```

Comment:
Since this became a simple `isContainedAt`, can we remove references to invalid links? This is really just a function that checks if a link is present inside an array of links.